lib/helpers/FormsUiHelper.ts

```diff
@@ -0,0 +1,436 @@
+      await expect(captionInput).toBeVisible();
+      await captionInput.fill("Test Caption " + (i + 1));
+    }
+  }
+
+  async ensureFormStylingSetting(){
+    await expect(this.formSettingStyling.locator('uui-label[title="styling"]')).toBeVisible();
+    const cssClassInput = this.formSettingStyling.locator("input[type = 'text']");
+    await expect(cssClassInput).toBeVisible();
+    await cssClassInput.fill("custom-css-class");
+    const disableDefaultStylesheetInput = this.formSettingStyling.locator('uui-toggle #slider');
+    await expect(disableDefaultStylesheetInput).toBeVisible();
+    await disableDefaultStylesheetInput.click();
+  }
+
+  async ensureFormValidationSetting(){
+    await expect(this.formSettingValidation.locator('uui-label[title="validation"]')).toBeVisible();
+    const requiredErrorMessageInput = this.formSettingValidation.locator("input[type = 'text']").nth(0);
+    await expect(requiredErrorMessageInput).toBeVisible();
+    await requiredErrorMessageInput.fill("Required error message");
+    const invalidErrorMessageInput = this.formSettingValidation.locator("input[type = 'text']").nth(1);
+    await expect(invalidErrorMessageInput).toBeVisible();
+    await invalidErrorMessageInput.fill("Invalid error message");
+
+    const showValidationSummaryInput = this.formSettingValidation.locator("uui-toggle #slider").nth(0);
+    await expect(showValidationSummaryInput).toBeVisible();
+    await showValidationSummaryInput.click();
+    const hideFieldValidationInput = this.formSettingValidation.locator("uui-toggle #slider").nth(1);
+    await expect(hideFieldValidationInput).toBeVisible();
+    await hideFieldValidationInput.click();
+
+    const markMandatoryFieldRadioInput = this.formSettingValidation.locator("uui-radio[value = 'MarkMandatoryFields']");
+    await expect(markMandatoryFieldRadioInput).toBeVisible();
+    await markMandatoryFieldRadioInput.click();
+
+    const indicatorInput = this.formSettingValidation.locator("input[type = 'text']").nth(2);
+    await expect(indicatorInput).toBeVisible();
+    await indicatorInput.fill("+");
+  }
+
+  async ensureFormAutocompleteSetting(){
+    await expect(this.formSettingAutocomplete.locator('uui-label[title="autocomplete"]')).toBeVisible();
+    const autocompleteAttributeRadioInput = this.formSettingAutocomplete.locator('uui-radio[value = "On"]');
+    await expect(autocompleteAttributeRadioInput).toBeVisible();
+    await autocompleteAttributeRadioInput.click();
+  }
+
+  async ensureFormModerationSetting(){
+    await expect(this.formSettingModeration.locator('uui-label[title="manualApproval"]')).toBeVisible();
+    const enablePostModerationAttributeToggleInput = this.formSettingModeration.locator('uui-toggle #slider');
+    await expect(enablePostModerationAttributeToggleInput).toBeVisible();
+    await enablePostModerationAttributeToggleInput.click();
+  }
+
+  async ensureFormFieldsDisplayedSetting(){
```

Comment:
Most of these ensure namings don't quite match what is actually happening in the methods.

Reply:
See my comment above for suggestion on naming.